base/summarysize.jl

```diff
@@ -148,6 +153,27 @@ function (ss::SummarySize)(obj::Array)
     return size
 end
 
+# FIXME Buffer GC
```

Comment:
Is this still pending?

Reply:
I forgot to go back and finish this once I figured out how to get malloced stuff working, but the results are still only as accurate as my assumptions about how jl_buffer_t is working behind the scenes.